src/components/UI/LanguageSelector/index.tsx

```diff
@@ -99,7 +99,7 @@ export default function LanguageSelector({
     <select
       data-testid="languageSelector"
       onChange={(event) => handleLanguageChange(event.target.value)}
-      className={className}
+      className={className + ' is-drop-down'}
```

Comment:
are these `is-drop-down` classes needed? I see they are added to select tags which are, by default, drop downs, isn't the `.settingSelect` selector enough?

Reply:
Needed for the cursor to change appropriately